docs/product-migration-guide/src/main/asciidoc/migration-execution-server-con.adoc

```diff
@@ -0,0 +1,11 @@
+[id='migration-execution-server-con']
+= Migrating {CENTRAL} as design environment and execution server
```

Comment:
This occurs in a few places. It just feels wrong not to have the article (as a design environment)

Reply:
I struggled BIG time with the titles in this doc. The reason I nixed the articles is that to me, the "an execution server" but just sounded weird and detached. "Central as a design env and an execution server". Dunno. Without the articles, just seemed less detached and streamlined. I'll think more on it though.